### ISSUE TITLE
Allow referencing parent with css-selector

### ISSUE THREAD

**Issue by Lewenhaupt** [mauricius/vue-draggable-resizable#223]
I think it would be great if we could select the parent based on css-selector instead of just a boolean.

**Comment by mauricius** [comment]
Yeah I tried to implement this once, without luck. I will try again in the future!

**Comment by Lewenhaupt** [comment]
Perhaps even better than css-selector could be with a vue ref to the HTMLElement?

**Comment by dvago** [comment]
Hi there, 
Thanks for this package

Just wanted to flag that I've got the same request, it would be great if we could get the parent via css-selector, there is another library which seems a sort of mirror that implements it, maybe could be a good example:

https://www.npmjs.com/package/vue-drag-resize

**Comment by h330894169** [comment]
+1

**Comment by zis-wtag** [comment]
+1
